src/index.js

```diff
@@ -1,2 +1,13 @@
 import './styles.css';
-import './modules/displauUI.js';
\ No newline at end of file
+import getData from './modules/getData.js';
+import addScore from './modules/addScoreToForm.js';
+
+addScore();
+
+document.querySelector('#refresh-btn').addEventListener('click', () => {
+  getData();
+});
```

Comment:
- [ ] Kindly review the Refresh button functionality for a better user experience. At the moment I clicked on it the list didn't update with the new scores, please 🙏🏽 fix it.